src/App.vue

```diff
@@ -1,12 +1,8 @@
 <template>
   <Theme :lang="lang" :theme="theme">
-    <WidgetMainSmall
-      title="Up To 60 months interest-free."
-      subtitle="Indicative payments. Ts&Cs apply."
-      button-primary-label="Explore"
-      button-close-label="Close"
+    <component
+      :is="currentWidget"
       :product-price="productPrice"
-      :lang="lang"
```

Comment:
I think we should still pass in `lang` so that it's always available to a widget